generator.yaml

```diff
@@ -1,26 +1,54 @@
 resources:
   Queue:
     unpack_attributes_map:
-      fields:
-        DelaySeconds:
-        MaximumMessageSize:
-        MessageRetentionPeriod:
-        KmsMasterKeyId:
-        KmsDataKeyReusePeriodSeconds:
-        Policy:
-        ReceiveMessageWaitTimeSeconds:
-        VisibilityTimeout:
-        FifoQueue:
-        ContentBasedDeduplication:
-        RedrivePolicy:
-        CreatedTimestamp:
-          is_read_only: true
-        LastModifiedTimestamp:
-          is_read_only: true
-        QueueArn:
-          is_read_only: true
+      set_attributes_single_attribute: false
       get_attributes_input:
         overrides:
           AttributeNames:
             values:
               - All
+    fields:
+      DelaySeconds:
+        is_attribute: true
+        type: string
+      MaximumMessageSize:
+        is_attribute: true
+        type: string
+      MessageRetentionPeriod:
+        is_attribute: true
+        type: string
+      KmsMasterKeyId:
+        is_attribute: true
+        type: string
+      KmsDataKeyReusePeriodSeconds:
+        is_attribute: true
+        type: string
+      Policy:
+        is_attribute: true
+        type: string
+      ReceiveMessageWaitTimeSeconds:
+        is_attribute: true
+        type: string
+      VisibilityTimeout:
+        is_attribute: true
+        type: string
+      FifoQueue:
+        is_attribute: true
+        type: string
+      ContentBasedDeduplication:
+        is_attribute: true
+        type: string
+      RedrivePolicy:
+        is_attribute: true
+        type: string
+      QueueArn:
+        is_attribute: true
+        is_arn: true
+        type: string
+      QueueName:
+        is_required: true
+        type: string
+      Tags:
+        compare:
+          is_ignored: true
```

Comment:
:+1: we'll handle updating tags using the `TagQueue`/`UntagQueue` API calls separately in a hook.